config/settings.yml

```diff
@@ -118,3 +118,6 @@ file_storage:
   default_root_path: <%= ::Rails.root %>/data/<%= ::Rails.env %>/
 
 rmd_convert_enabled: false
+
+resque:
+  permitted_hosts: [".teach.cs.toronto.edu"]
```

Comment:
This isn't an appropriate default value. I imagine `localhost` is better; we can override this setting in a custom production environment.